### PR TITLE
remove Jakarta XML RPC testing

### PR DESCRIPTION
**Pull request by gurunrao**
Signed-off-by: Gurunandan <gurunandan.rao@oracle.com>

### CODE REVIEW

File: install/jakartaee/bin/build.xml [hunk 1]
```diff
@@ -55,10 +55,8 @@
                   com/sun/ts/tests/jms,
                   com/sun/ts/tests/jsonp,
                   com/sun/ts/tests/jsonb,
-                  com/sun/ts/tests/jaxrpc,
                   com/sun/ts/tests/jaxws/common,
                   com/sun/ts/tests/jaxws/wsi/constants,
-                  com/sun/ts/tests/webservices,
```
Comment (alwin-joseph) [inline install/jakartaee/bin/build.xml:61]: Just checking, are all the tests in webservices depended on jaxrpc ?

**Comment by alwin-joseph** [comment]
@gurunrao Thanks for this change.
-Can you please confirm all the tests under src/com/sun/ts/tests/webservices are depended on jaxrpc.
-Are there any tests removed other than src/com/sun/ts/tests/webservices, src/com/sun/ts/tests/internal/implementation/sjsas/jaxrpc and src/com/sun/ts/tests/jaxrpc ?

**Comment by gurunrao** [comment]
> @gurunrao Thanks for this change.
> -Can you please confirm all the tests under src/com/sun/ts/tests/webservices are depended on jaxrpc.
> -Are there any tests removed other than src/com/sun/ts/tests/webservices, src/com/sun/ts/tests/internal/implementation/sjsas/jaxrpc and src/com/sun/ts/tests/jaxrpc ?

thanks @alwin-joseph  for the review.
> src/com/sun/ts/tests/webservices has only jaxrpc services
> along with tests mentioned above - we have also removed jaxrpc webservice test at ejb/ee/webservices/allowedmethodstest

**Comment by brideck** [comment]
Apologies for the really old bump.  I was looking through -javaee_optional tests in `ejb` and was wondering why we left anything at all in ejb/ee/webservices/allowedmethodstest.  It looks like the test class, etc. was removed, but we still have some application gorp here that gets built for some reason.

**Pull request merged**